web/src/condense.js

```diff
@@ -281,4 +281,12 @@ export function initialize() {
         e.stopPropagation();
         e.preventDefault();
     });
+
+    $("#message_feed_container").on("click", ".message_condenser, .message_expander", function () {
+        // Clicking on the "Show more" or "Show less"
+        // buttons should focus on the message.
+        const $row = $(this).closest(".message_row");
+        const id = rows.id($row);
+        message_lists.current.select_id(id);
+    });
```

Comment:
I don't like this at all -- we should add the code to the existing click handlers for those two cases. One reason that might be important is that the others use `e.stopPropagation();`, so we can't count on this code running as written.

Reply:
As I know, when we use e.stopPropagation(); in an event listener, it prevents the event from being propagated to the parent elements in the DOM, but it doesn't prevent other event listeners on the same element from being triggered. So, the second event listener on the same element would still get triggered.

Reply:
So this will work, unless we use [e.stopImmediatePropagation](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopImmediatePropagation).

Reply:
I agree that we already have event listeners for both condense and expand, but I found it helpful to put this logic into a separate listener. That way, we won't lose the logic somewhere or accidentally delete it.

Reply:
I think it's less confusing in this case to have a single handler for a given click -- it makes it much easier to reason about.